Makefile

```diff
@@ -58,3 +58,6 @@ check-internal-links: clean build link-checker-setup run-link-checker
 
 check-all-links: clean build link-checker-setup
 	bin/htmltest --conf .htmltest.external.yml
+
+spellcheck:
```

Comment:
this is not called from CI anywhere

Reply:
Now `ci-test.yml` is calling spellcheck and exits if a misspelling is found.